src/Form/Mixin/Value.php

```diff
@@ -136,6 +136,20 @@ protected function setDefault(mixed $default = null): void
 		$this->default = $default;
 	}
 
+	/**
+	 * Submits a new value for the field.
+	 * Fields can overwrite this method to provide custom
+	 * submit logic. This is useful if the field component
+	 * sends data that needs to be processed before being
+	 * stored.
+	 *
+	 * @since 5.0.0
+	 */
+	public function submit(mixed $value): static
+	{
+		return $this->fill($value);
```

Comment:
`Understanding:` E.g. with the picker fields, this would be the place where we transform the big data array to simple list of (UU)IDs?

Reply:
Yes, exactly. We basically have two ways now: 

**::fill:**
values come from storage, will be used for field components and might need to be transformed

**::submit** 
values come from form submissions and will eventually go into storage and they might need to be transformed.